core/src/com/mygdx/game/GameEngine.java

```diff
@@ -146,7 +146,13 @@ public float addSingleResult(LocalRayResult rayResult, boolean normalInWorldSpac
 	private Array<GameModel> dynamicModels = new Array<GameModel>();
 	private Bits visibleLayers = new Bits();
 
+	// FIXME
+	// Ugly hack to access the engine from anywhere
+	public static GameEngine engine;
+
```

Comment:
Perhaps, though GameEngine is supposed to be a singleton anyway, so if we make it final it should work fine.